ce_main_app/native/scsi.cpp

```diff
@@ -665,6 +666,25 @@ void Scsi::SCSI_Inquiry(void)
         return;
     }
 
+    if(devInfo[acsiId].attachedMediaIndex != -1) {
+        int type = attachedMedia[devInfo[acsiId].attachedMediaIndex].hostSourceType;
+        switch(type) {
+        case SOURCETYPE_IMAGE:
+          memcpy(type_str, " IMG", 4);
+          break;
+        case SOURCETYPE_IMAGE_TRANSLATEDBOOT:
+          memcpy(type_str, "BOOT", 4);

```

Comment:
Feel free to improve my patch :) I think also device id could be moved from index 27 to 24 or 25 to leave more space for that "costomized" name

assuming you had defined a remote repository to mine :
$ git remote add nanard https://github.com/miniupnp/ce-atari.git
$ git fetch --all --prune
you create a new branch in your repository this way :
$ git checkout -b inquiry_branch nanard/dev_name

when your branch will be merged into master, the PR will be automatically closed !